### PR TITLE
Add several Python api-doc examples

### PR DESCRIPTION
**Pull request by tomasvr**
Added examples for several Python API functions.

### CODE REVIEW

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
We should not add python documentation to the base API definitions. These definitions are visibile on all languages for which we have bindings and it does not make sense to have Python doctests in a C++ documentation, for example

**Comment by tomasvr** [comment]
I see, sorry I was not aware of that. Is there any way I can make the examples appear only in the Python API docs? Then I could perhaps try to amend it. Thank you for your feedback.

**Comment by mihaimaruseac** [comment]
See 8582a58

**Comment by gbaned** [comment]
@tomasvr Can you please check reviewer comments and keep us posted. Thanks!

**Comment by tomasvr** [comment]
@gbaned I removed the examples from the general API for now. I'm currently preoccupied with new courses so I have to leave the other examples for now.

File: tensorflow/python/ops/linalg_ops.py [hunk 1]
```diff
@@ -410,6 +410,14 @@ def eigvals(tensor, name=None):
   eigenvectors are used to compute the gradient w.r.t. the eigenvalues. See
   _SelfAdjointEigV2Grad in linalg_grad.py.
 
+  For example:
+  ```
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/linalg_ops.py:414]:
```suggestion

```

File: tensorflow/python/ops/linalg_ops.py [hunk 2]
```diff
@@ -410,6 +410,14 @@ def eigvals(tensor, name=None):
...
+  >>> a = tf.constant([[1,2],[3,4]], dtype=tf.float32)
+  >>> tf.linalg.eigvalsh(a)
+  <tf.Tensor: shape=(2,), dtype=float32, numpy=
+  array([-0.8541021,  5.854102 ], dtype=float32)>
+  ```
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/linalg_ops.py:419]:
```suggestion
```

**Review by mihaimaruseac — COMMENTED** [review]
The triple backticks (` ``` `) are not needed. https://www.tensorflow.org/community/contribute/docs_ref

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by mihaimaruseac** [comment]
Can you fix the Ubuntu CPU test, please?

**Comment by gbaned** [comment]
@tomasvr Can you please check @mihaimaruseac comments and keep us posted. Thanks!

**Comment by tomasvr** [comment]
I am not sure what is causing the Ubuntu CPU to fail. I simply copied the results for the example calculation straight from the actual output of my terminal itself.

**Comment by tomasvr** [comment]
@gbaned I believe it's fixed now.

**Comment by gbaned** [comment]
@tomasvr Can you please address Ubuntu Sanity errors? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Pull request closed without merging**